services/minespace-web/src/components/common/QuestionsSidebar.js

```diff
@@ -1,17 +1,19 @@
 import React from "react";
+import * as Strings from "@/constants/strings";
 
 const QuestionSidebar = () => (
   <div className="sidebar-block">
     <div className="sidebar-block-title">
-      <h2 className="side-bar-title">Questions? Issues?</h2>
+      <h2 className="side-bar-title">Questions?</h2>
     </div>
     <div className="sidebar-block-content">
       <p>
-        We encourage your feedback and would like to hear from you. If you would like to get in
-        touch with us, send us an email to{" "}
-        <a className="underline" href="mailto:MDS@gov.bc.ca">
-          mds@gov.bc.ca
+        We encourage your feedback and would like to hear from you. If you have any questions,
+        feedback, or other concerns, please send us an email at&nbsp;
+        <a className="underline" href={`mailto:${Strings.MDS_EMAIL}`}>
+          {Strings.MDS_EMAIL}
         </a>
+        .
```

Comment:
I thought this looked/sounded nicer, but no worries if you want me to revert it to the previous text.

Reply:
I know have access to the MineSpace mockups which has the updated text to use.

Reply:
A lot has changed for this component/the home page in general so we can address this text again in a future ticket.